Source/ablastr/utils/SignalHandling.cpp

```diff
@@ -140,6 +152,10 @@ SignalHandling::InitSignalHandling()
 void
 SignalHandling::CheckSignals()
 {
+    // Do not perform MPI communication if no signal action is configured
```

Comment:
The comment here threw me off - I was going to remark that this check should come after the testing of signal flags, lest signal handling be disabled in the single-process case. I had to think to realize that this is itself a check whether any signal handling is enabled. Please reword the comment

Reply:
Thx, rephrased :)